src/App.tsx

```diff
@@ -1,73 +1,106 @@
-import React from 'react';
+import React, { useMemo, useState } from 'react';
+import debounce from 'lodash.debounce';
 import './App.scss';
+import cn from 'classnames';
 import { peopleFromServer } from './data/people';
+import { Person } from './types/Person';
 
 export const App: React.FC = () => {
-  const { name, born, died } = peopleFromServer[0];
+  const [query, setQuery] = useState('');
+  const [appliedQuery, setAppliedQuery] = useState('');
+  const [selectedPerson, setSelectedPerson] = useState<Person | null>(null);
+  const [dropDown, setDropDown] = useState(false);
```

Comment:
```suggestion
  const [isFocused, setIsFocused] = useState(false);
```
It describes value better
just FYI